draft-ietf-scitt-architecture.md

```diff
@@ -243,8 +231,6 @@ Registration:
 Registration Policy:
 
 : the pre-condition enforced by the Transparency Service before registering a Signed Statement, based on information in the non-opaque header and metadata contained in its COSE Envelope.
-A Transparency Service MAY implement any range of policies that meets their needs.
-However a Transparency Service can not alter the contents of the Signed Statements.
```

Comment:
Has it been relocated elsewhere in the document?

Reply:
Yes, below